### PR TITLE
Task 1.1 Two A duodecimal range checker

### PR DESCRIPTION
**Pull request by stoyankisyov**
I solved it using switch-case. In case I'm not allowed to use switch-case for current tasks, it can be solved by using array with elements {"0", "1", ..., "A", "B"} -> result = array[remainder] + result;. 
Decided to leave it with switch-case because the chance for potential bugs is lesser, since array's elements can be easily accessed by index and get it's values changed + easier for reading and maintaining.

### CODE REVIEW

File: TwoADuodecimalRangeChecker/Program.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
+﻿namespace TwoADuodecimalRangeChecker
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
```
Comment (IgorKheidorov) [inline TwoADuodecimalRangeChecker/Program.cs:6]: a can be less then b , is a tasks no limitations to the order of input values. One of the numbers can be negative. Please process this case

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Totally good, see comments

File: TwoADuodecimalRangeChecker/Program.cs (hunk 1 above)
Comment (stoyankisyov) [reply TwoADuodecimalRangeChecker/Program.cs:6]: changes done, now works with negative numbers as well + a and b order doesn't affect the behavior of the application

**Review by IgorKheidorov — APPROVED** [review]
Good for now)

**Pull request merged**